### PR TITLE
feat(remotepins): add Functionland Fula

### PR DESCRIPTION
**Pull request by ehsan6sha**
Functionland Fula is an ipfs compatible DePIN network of 900 (as of 28/06/2024) individually owned and globally distributed nodes. The pinning service report is available at: [https://github.com/functionland/pinning-service/blob/main/api.cloud.fx.land.md](https://github.com/functionland/pinning-service/blob/main/api.cloud.fx.land.md)
Information on Functionland is avaialbe at https://fx.land and also https://www.crunchbase.com/organization/functionland/signals_and_news/timeline

### CODE REVIEW

**Comment by ehsan6sha** [comment]
Hey. Any feedback on this?

**Comment by SgtPooki** [comment]
@ehsan6sha hey, thanks for submitting this PR and throwing together a pinning service compliance report! we are busy prepping for IPFS Camp but the team will chat about getting this in the webui.

File: src/constants/pinning.js
Comment (lidel) [inline src/constants/pinning.js:39]: @ehsan6sha what is the ETA for general availability?

The docs at https://docs.fx.land/pinning-service/ipfs-pinning-service-api#authentication state it is invite-only, and https://cloud.fx.land/ is a broken link.

WebUI is shipping inside IPFS Desktop, and as bare minimum for reviewing this PR, a regular user needs to be able to use it.

**Review by lidel — CHANGES_REQUESTED** [review]

File: src/constants/pinning.js
Comment (ehsan6sha) [reply src/constants/pinning.js:39]: Thank you @lidel for mentioning that . We were planning to officially announce the release later, but I made it live now. the website and instructions are updated and easy to use for anyone:
https://cloud.fx.land/

**Comment by ehsan6sha** [comment]
Hey. Please let me know if anything else needs modification before being able to merge 
Thnak you

**Comment by ehsan6sha** [comment]
Hi, Just checking on this PR

**Comment by lidel** [comment]
Apologies for slow reply, this project is at the end of our release pipelines and not triaged often.

My understanding is the inclusion bar is to pass and be added to continously running  https://github.com/ipfs-shipyard/pinning-service-compliance tests.

You can follow what Scaleway did (+share test token with us):
1. https://github.com/ipfs-shipyard/pinning-service-compliance/issues/289
2. https://github.com/ipfs-shipyard/pinning-service-compliance/pull/333

That being said, there seems to be a [problem with updating conformance report website](https://github.com/ipfs-shipyard/pinning-service-compliance/issues/289#issuecomment-2282751412) at the moment, and to not block on that, I think as long   `npx @ipfs-shipyard/pinning-service-compliance -s <your_service_endpoint> <auth_token>` passes, we can merge and release this.

@ehsan6sha to expedite, are you able to run test and post results here? Or share auth token with us on Slack DM.

**Comment by ehsan6sha** [comment]
> Apologies for slow reply, this project is at the end of our release pipelines and not triaged often.
> 
> My understanding is the inclusion bar is to pass and be added to continously running https://github.com/ipfs-shipyard/pinning-service-compliance tests.
> 
> You can follow what Scaleway did (+share test token with us):
> 
> 1. [Add Scaleway IPFS Pinning for compliance ipfs-shipyard/pinning-service-compliance#289](https://github.com/ipfs-shipyard/pinning-service-compliance/issues/289)
> 2. [feat: add scaleway pinning service ipfs-shipyard/pinning-service-compliance#333](https://github.com/ipfs-shipyard/pinning-service-compliance/pull/333)
> 
> That being said, there seems to be a [problem with updating conformance report website](https://github.com/ipfs-shipyard/pinning-service-compliance/issues/289#issuecomment-2282751412) at the moment, and to not block on that, I think as long `npx @ipfs-shipyard/pinning-service-compliance -s <your_service_endpoint> <auth_token>` passes, we can merge and release this.
> 
> @ehsan6sha to expedite, are you able to run test and post results here? Or share auth token with us on Slack DM.

The report is available here: https://github.com/functionland/pinning-service/blob/main/api.cloud.fx.land.md

I also shared a test token in slack

**Review by lidel — APPROVED** [review]
Thank you!

File: src/constants/pinning.js
Comment (lidel) [inline src/constants/pinning.js:37]:
```suggestion
    icon: 'https://dweb.link/ipfs/bafybeidgnnvgm6i3pfzhjakgzcdwdbvwtmkkx7vzeiyvtk2b4oxis53vhm?filename=functionland.png',
```

File: src/constants/pinning.js
Comment (lidel) [inline src/constants/pinning.js:37]: Just realized IPFS version is 5000x5000 pixels in size for some reason.
@ehsan6sha  do you have SVG by any chance? If not, we should switch to 512x512.

**Review by lidel — CHANGES_REQUESTED** [review]

File: src/constants/pinning.js
Comment (lidel) [reply src/constants/pinning.js:37]: Provisionally, replaced 192k PNG with 7k SVG:
```suggestion
    icon: 'https://dweb.link/ipfs/QmWYEmdYq9Ry2xtb69oZSPXb8Aos24kWdVecsT3txVe38E?filename=functionland.svg',
```

Let's see how it looks.

**Review by lidel — APPROVED** [review]
SVG looks fine, shipping.

**Pull request merged**
